docs/data-cookbooks/cnm-workflow.md

```diff
@@ -102,6 +102,7 @@ state_machine_definition = <<JSON
           "cma": {
             "event.$": "$",
             "task_config": {
+              "collection": "{$.meta.collection}",
```

Comment:
This workflow is a partial example that doesn't show using `CnmResponseFail` and thus doesn't demonstrate how to setup the double failure branches we're using in our actual CNM example workflow to compensate for how `CnmResponse` is written. Does that matter?

Reply:
Hmmm not sure. I do know we're going to talk to PODAAC about some changes to these, so maybe we could work with them on that in 20.1 so we don't have to do so much there.

Reply:
Yeah, I have already flagged the issue for them that requires the double failure branches and they created a ticket to address it:

https://bugs.earthdata.nasa.gov/browse/PCESA-1446

So perhaps we can leave this as is for now